doc/themes/the-things-stack/layouts/shortcodes/openapi/openapi.html

```diff
@@ -0,0 +1,163 @@
+{{ $openapi := .Get "file" | index .Site.Data.api }} {{ $tag := .Get "service"
+}}
+<div>
+  <div class="api-page-header">
+    <h3>The Things Stack HTTP (REST) API Reference</h3>
+    <p>Version: <code> {{ $openapi.info.version }} </code></p>
+  </div>
+  <div class="api-tags-table">
+    <table>
+      <thead>
+        <tr>
+          <th>Service</th>
+          <th>Description</th>
+        </tr>
+      </thead>
+      <tbody>
+        {{ range $tagDetails := $openapi.tags }}
```

Comment:
This needs to be sorted

Reply:
What should it be sorted on?
It's currently sorted alphabetically and there aren't really options to set sorting: https://github.com/grpc-ecosystem/grpc-gateway/blob/main/protoc-gen-openapiv2/options/openapiv2.proto#L587

Reply:
Then I think I put the comment on the wrong line, because what I meant is that the table with services is not sorted alphabetically.